flac needs AM_PATH_XMMS macro to build the plugin for XMMS, but it's not
shipped with it. This patch adds the m4 file to the package.

Index: flac-1.1.3/configure.in
===================================================================
--- flac-1.1.3.orig/configure.in
+++ flac-1.1.3/configure.in
@@ -212,9 +212,6 @@ AC_HELP_STRING([--disable-xmms-plugin], 
 	no)  enable_xmms_plugin=false ;;
 	*) AC_MSG_ERROR(bad value ${enableval} for --enable-xmms-plugin) ;;
 esac],[enable_xmms_plugin=true])
-if test "x$enable_xmms_plugin" != xfalse ; then
-	AM_PATH_XMMS(0.9.5.1, , AC_MSG_WARN([*** XMMS >= 0.9.5.1 not installed - XMMS support will not be built]))
-fi
 AM_CONDITIONAL(FLaC__HAS_XMMS, test -n "$XMMS_INPUT_PLUGIN_DIR")
 
 dnl check for ogg library
